.travis.yml

```diff
@@ -1,27 +1,82 @@
 language: go
 
-matrix:
-  include:
-  -  os: linux
-     sudo: required
-     services:
-       - docker
-  -  os: osx
-
 go:
   - 1.13.x
 
 notifications:
   email: false
 
-install:
-  # Ideally, the (brew update) should not be necessary and Travis would have fairly
-  # frequenstly updated OS images; that’s not been the case historically.
-  # In particular, explicitly unlink python@2, which has been removed from Homebrew
-  # since the last OS image build (as of July 2020), but the Travis OS still
-  # contains it, and it prevents updating of Python 3.
-  - if [[ "$TRAVIS_OS_NAME" == "osx" ]]; then brew update && brew unlink python@2 && brew install gpgme ; fi
-
-script:
-  - if [[ "$TRAVIS_OS_NAME" == "osx" ]];   then hack/travis_osx.sh ; fi
-  - if [[ "$TRAVIS_OS_NAME" == "linux" ]]; then make vendor && ./hack/tree_status.sh && make local-cross && make check ; fi
+env:
+  global:
+    - REGISTRY=quay.io
+    - MA_TAG=ma
+
+jobs:
+  allow_failures:
+  - if: arch in (s390x, ppc64le)
+  include:
+    - stage: local-build
+      <<: *local-build
+      name: local build for osx
+      env: SOURCE_TYPE=none
+      os: osx
+      install:
+        # Ideally, the (brew update) should not be necessary and Travis would have fairly
+        # frequenstly updated OS images; that�~@~Ys not been the case historically.
```

Comment:
Typo ("frequently", no "s"). And, weird localization characters. Please make sure your editor is set to UTF-8.

Reply:
hey, I just copied it from existing text :) fixed